shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -770,14 +770,21 @@ public ExclusiveAppComponent<Activity> getExclusiveAppComponent() {
     return delegate;
   }
 
+  // We track the attached context so we can unregister component callbacks when detaching.
+  private Context attachedContext;
```

Comment:
I suspect you don't need to do this. I think just getContext is fine, and if the context changes because the fragment gets moved to a different activity or something, onAttach should be called (and get registered to another callback)